src/App.tsx

```diff
@@ -1,61 +1,84 @@
+import React, { FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('0');
+  const [todolist, setTodolist] = useState(todosFromServer);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const handleTitleChange = (value: string) => {
+    const validateTitle = value.split('').filter(char => {
+      return char.match(/^[A-Za-z]*$/) || char.match(/\s/);
+    });
+
+    setTitle(validateTitle.join(''));
+  };
+
+  const handleOnSubmit = (eventsubmit: FormEvent) => {
+    eventsubmit.preventDefault();
+    setTitleError(!title.trim());
+    setUserError(user === '0');
```

Comment:
```suggestion
if(user === '0'){
  setUserError(true);
} 
```

it's better to wrap this into `if` to avoid calling this function redundantly